### PR TITLE
refactor!: remove blob sign/verify related contents

### PR DESCRIPTION
**Pull request by Two-Hearts**
As discussed, this PR removes the blob sign/verify related contents for v1.2.0 release.

This PR targets on branch `release-1.2` ONLY.

### CODE REVIEW

File: specs/commandline/policy.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Description
 
-As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [OCI trust policy specification and examples](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#oci-trust-policy).
+As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [trust policy specification and examples](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#trust-policy).
```
Comment (JeyJeyGao) [inline specs/commandline/policy.md:5]: The link is 404 page not fund

Comment (Two-Hearts) [reply specs/commandline/policy.md:5]: Yes, I'm actually aware of this, but since we will release the `v1.1.0` spec recently, I kept it.

Comment (Two-Hearts) [reply specs/commandline/policy.md]: Updated to `v1.1.0-rc.1` to make the links work. We will need to update them to `v1.1.0` once the stable version of the spec is released.

**Review by JeyJeyGao — COMMENTED** [review]
LGTM. Pending on dependencies.

File: go.mod [hunk 2]
```diff
@@ -32,3 +32,5 @@ require (
 	golang.org/x/sync v0.6.0 // indirect
 	golang.org/x/sys v0.22.0 // indirect
 )
+
+replace github.com/notaryproject/notation-go => github.com/Two-Hearts/notation-go v0.0.0-20240815061530-d5e876aecb1d
```
Comment (shizhMSFT) [inline go.mod:36]: Leave a comment as a reminder for dependency update.

File: test/e2e/plugin/go.mod [hunk 3]
```diff
@@ -29,3 +29,5 @@ require (
 	golang.org/x/sync v0.6.0 // indirect
 	oras.land/oras-go/v2 v2.5.0 // indirect
 )
+
+replace github.com/notaryproject/notation-go => github.com/Two-Hearts/notation-go v0.0.0-20240815061530-d5e876aecb1d
```
Comment (shizhMSFT) [inline test/e2e/plugin/go.mod:33]: Leave a comment as a reminder for dependency update.

File: specs/commandline/inspect.md [hunk 4]
```diff
@@ -24,13 +24,10 @@ Upon successful execution, both the digest of the signed artifact and the digest
         └── <signed artifact>
 ```
 
-> [!NOTE]
-> This command is for inspecting signatures associated with OCI artifacts only. Use `notation blob inspect` command for inspecting signatures associated with arbitrary blobs.
-
```
Comment (shizhMSFT) [inline specs/commandline/inspect.md:29]: This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

File: specs/commandline/policy.md [hunk 5]
```diff
@@ -2,9 +2,9 @@
...
 
-As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [OCI trust policy specification and examples](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#oci-trust-policy).
+As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [trust policy specification and examples](https://github.com/notaryproject/specifications/blob/v1.1.0-rc.1/specs/trust-store-trust-policy.md#trust-policy).
 
-The `notation policy` command provides a user-friendly way to manage trust policies for signed OCI images. It allows users to show trust policy configuration, import/export a trust policy configuration file from/to a JSON file. Users who want to manage trust policies for signed arbitrary blobs, please refer to `notation blob policy` command.
```
Comment (shizhMSFT) [inline specs/commandline/policy.md:7]: This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

File: specs/commandline/sign.md [hunk 6]
```diff
@@ -19,8 +19,6 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed <registry>/<repository>@<digest>
 ```
 
-NOTE: This command is for signing OCI artifacts only. Use `notation blob sign` command for signing arbitrary blobs.
```
Comment (shizhMSFT) [inline specs/commandline/sign.md:22]: This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

File: specs/commandline/verify.md [hunk 7]
```diff
@@ -25,9 +25,6 @@ The artifact was signed with the following user metadata.
 KEY    VALUE
 <key>  <value>
 ```
-> [!NOTE]
-> This command is for verifying OCI artifacts only. Use `notation blob verify` command for verifying arbitrary blobs.
```
Comment (shizhMSFT) [inline specs/commandline/verify.md:29]: This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

File: specs/notation-cli.md [hunk 8]
```diff
@@ -5,8 +5,7 @@ This spec contains reference information on using notation commands. Each comman
 ## Notation Commands
 
 | Command                                     | Description                                                            |
-| ------------------------------------------- | ---------------------------------------------------------------------- |
-| [blob](./commandline/blob.md)               | Sign, verify and inspect singatures associated with blobs              |
```
Comment (shizhMSFT) [inline specs/notation-cli.md:9]: This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

File: specs/commandline/verify.md
Comment (shizhMSFT) [inline specs/commandline/verify.md:1]: Line 61 has `trustpolicy.oci.json` in it.

**Review by shizhMSFT — COMMENTED** [review]
Cross-reviewed with
- https://github.com/notaryproject/notation/compare/v1.1.1..Two-Hearts:notation:release
- https://github.com/notaryproject/notation/compare/v1.1.0..Two-Hearts:notation:release

File: specs/commandline/verify.md
Comment (Two-Hearts) [reply specs/commandline/verify.md:1]: Removed.

File: specs/commandline/inspect.md (hunk 4 above)
Comment (Two-Hearts) [reply specs/commandline/inspect.md:29]: @shizhMSFT Created issue to track it:
https://github.com/notaryproject/notation/issues/1012 @JeyJeyGao

File: specs/commandline/policy.md (hunk 5 above)
Comment (Two-Hearts) [reply specs/commandline/policy.md:7]: Tracking: https://github.com/notaryproject/notation/issues/1012

File: specs/commandline/sign.md (hunk 6 above)
Comment (Two-Hearts) [reply specs/commandline/sign.md:22]: Tracking: https://github.com/notaryproject/notation/issues/1012

File: specs/notation-cli.md (hunk 8 above)
Comment (Two-Hearts) [reply specs/notation-cli.md:9]: Tracking: https://github.com/notaryproject/notation/issues/1012

File: go.mod (hunk 2 above)
Comment (Two-Hearts) [reply go.mod]: Updated.

File: test/e2e/plugin/go.mod (hunk 3 above)
Comment (Two-Hearts) [reply test/e2e/plugin/go.mod]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by vaninrao10 — COMMENTED** [review]
I see the OCI is removed in the inspect.md for the artifacts, but in policy .md we have retained the OCI artifact, dont we think there is a mismatch here "of signature verification workflow of signed OCI artifacts" ? Will it confuse the reader.

**Pull request merged**